projects/igniteui-angular/src/lib/snackbar/snackbar.component.html

```diff
@@ -2,7 +2,9 @@
     {{ textMessage }}
     <ng-content></ng-content>
 </div>
-<ng-content *ngIf="buttonTemplate" select="[igxButton]"></ng-content>
-<button *ngIf="!buttonTemplate && actionText" igxButton (click)="triggerAction()">
+<div #customButton class="custom-btn">
```

Comment:
Have you tried using `ng-container` instead of `div`? Then it shouldn't interfere with the DOM and the result would be identical with default and custom button. You would also not need the class.